### ISSUE TITLE
Broken Lockfile Action

### ISSUE THREAD

**Issue by rcomer** [SciTools/iris#4606]
## 📰 Custom Issue

The lockfile update action appears to be broken.
https://github.com/SciTools/iris/actions/workflows/refresh-lockfiles.yml

**Comment by trexfeathers** [comment]
We're thinking of removing the offending lines and simply instructing people to not update the bot's branch (whenever that might be necessary), but to replace with their own PR incorporating the bot's proposed changes.

**Comment by rcomer** [comment]
Not updating the bot's branch would also avoid issues when the action runs and replaces its existing commit.  See https://github.com/SciTools/iris/pull/4428#issuecomment-980536370

**Comment by jamesp** [comment]
Is the problem that dependabot updated the script version from 5.1.0 to 6? https://github.com/SciTools/iris/blame/c83722b0b5241958309c2843d9d715f012faa48b/.github/workflows/refresh-lockfiles.yml#L40

**Comment by rcomer** [comment]
@jamesp no, it was already broken with 5.1 https://github.com/SciTools/iris/runs/5164038812?check_suite_focus=true
